src/cases/containers/case-hearings/case-hearings.component.ts

```diff
@@ -23,8 +23,10 @@ export class CaseHearingsComponent implements OnInit, OnDestroy {
   public upcomingStatus: EXUISectionStatusEnum = EXUISectionStatusEnum.UPCOMING;
   public pastAndCancelledHearings$: Observable<HearingListViewModel[]>;
   public pastAndCancelledStatus: EXUISectionStatusEnum = EXUISectionStatusEnum.PAST_AND_CANCELLED;
+  public hearingState$: Observable<fromHearingStore.State>;
   public hearingsActions: Actions[] = [Actions.READ];
   public userRoles$: Observable<string[]>;
+  public sub: Subscription;
```

Comment:
is this sub needed?

Reply:
I have removed this code which is not required.